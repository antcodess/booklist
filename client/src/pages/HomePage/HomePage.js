import React from 'react';
import ImageSlider from '../../components/ImageSlider/ImageSlider';
import Directory from '../../components/Directory/Directory';

export const HomePage = () => {
  return (
    <React.Fragment>
      <ImageSlider />
      <Directory />
    </React.Fragment>
  );
};

export default HomePage;