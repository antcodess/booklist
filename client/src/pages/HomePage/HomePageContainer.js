import { connect } from 'react-redux';
import { compose } from 'redux';
import { createStructuredSelector } from 'reselect';
import { selectIsProductFetching } from '../../redux/shop/shop.selectors';

import Spinner from '../../components/Spinner/Spinner';
import HomePage from './HomePage';

const mapStateToProps = createStructuredSelector({
  isLoading: selectIsProductFetching,
});

const HomePageContainer = compose(
  connect(mapStateToProps),
  Spinner
)(HomePage);

export default HomePageContainer;