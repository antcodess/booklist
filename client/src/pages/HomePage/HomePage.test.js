import React from 'react';
import { shallow } from 'enzyme';
import { HomePage } from './HomePage';

describe('HomePage Component', () => {
  let wrapper;
  
  beforeEach(() => {
    wrapper = shallow(<HomePage />);
  });

  it('should render HomePage Component', () => {
    expect(wrapper).toMatchSnapshot();
  });
});