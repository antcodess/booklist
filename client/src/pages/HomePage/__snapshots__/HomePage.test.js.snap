// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage Component should render HomePage Component 1`] = `ShallowWrapper {}`;
